src/one_dragon_qt/widgets/setting_card/combo_box_setting_card.py

```
@@ -82,6 +84,22 @@ def _initialize_options(self, options_enum: Optional[Iterable[Enum]], options_li
                 self._opts_list.append(opt_item)
                 self.combo_box.addItem(opt_item.ui_text, userData=opt_item.value)
 
+    def showEvent(self, event: QEvent) -> None:
+        super().showEvent(event)
+        if self._content_shrink:
+            self._shrink_content()
+
+    def _shrink_content(self) -> None:
+        """下拉框变宽时收窄说明文字，右边界不动、下拉框文字完整显示"""
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)